src/views/app/detail-panel/edit/edit-attachments-block.jsx

```diff
@@ -102,7 +102,11 @@ function Attachment({ filename, size, link, editor, part, iconColors }) {
 				}
 			})
 		);
-	}, [editor, dispatch, part]);
+		throttledSaveToDraft({
```

Comment:
Are you sure you need to throttle this function?

Reply:
@nubsthead what do you mean ? It's a function to do the save draft request and as per the previous ticket regarding auto save draft request we need to do it for each change

Reply:
From what i understand that function will start a timer and it will save the draft at the end of the timer. I think in thise case we don't need to set a timer to save the draft but to run it as soon as user clicks on it. Do you see any issue on that?

Reply:
We'll this is the beauty of this function ,if I remove it then the draft auto save will be trigger for each character change so for a word abhishek it will create 8 saveDraft request.By implementing the clearTimeout and setTimeout in the above manner it will only do the request if it is not clearTimeout i.e once user stops typing.

Reply:
I agree, but this field does not get triggered very often, do you think it needs to be delayed?

Reply:
Yes,Let's take an example,
You are a User who's typing speed is very fast and you have to type a long message in body which have around 300 words and each word will have minimum 3 characters .So without that function the min save draft request will be 300*3 = 900 times 😁 but with that function once user stops typing the request will be triggered which means the saveDrafts request will be around 300-400 😃